api/v1beta1/cloudstackmachine_webhook.go

```diff
@@ -84,15 +84,15 @@ func (r *CloudStackMachine) ValidateUpdate(old runtime.Object) error {
 	}
 	oldSpec := oldMachine.Spec
 
-	errorList = webhookutil.EnsureBothFieldsAreEqual(r.Spec.Offering.ID, r.Spec.Offering.Name, oldSpec.Offering.ID, oldSpec.Offering.Name, "offering", errorList)
+	// errorList = webhookutil.EnsureBothFieldsAreEqual(r.Spec.Offering.ID, r.Spec.Offering.Name, oldSpec.Offering.ID, oldSpec.Offering.Name, "offering", errorList)
```

Comment:
@davidjumani, once rejosh confirms removal of these is OK, please completely delete these lines.

Reply:
Will create a separate PR to do so as well as fix any tests as well

Reply:
@davidjumani 
Other PR went through. Please merge and resolve conflicts.